### PR TITLE
TT 155 consume user info

### PR DESCRIPTION
**Pull request by thegreatyamori**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[1207/120956.471:ERROR:directory_reader_win.cc(43)] FindFirstFile: El sistema no puede encontrar la ruta especificada. (0x3)
```
Comment (Angeluz-07) [inline debug.log:1]: Remove this file.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -161,9 +162,10 @@ const maskConfig: Partial<IConfig> = {
       TechnologyEffects,
       ProjectTypeEffects,
       EntryEffects,
+      UsersEffects,
```
Comment (scastillo-jp) [inline src/app/app.module.ts:165]: You need to keep these two imports? `UsersEffects`  `UserEffects`

File: src/app/modules/shared/components/user/user.component.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { AzureAdB2CService } from '../../../login/services/azure.ad.b2c.service';
 
+
```
Comment (scastillo-jp) [inline src/app/modules/shared/components/user/user.component.ts:4]: delete this line

File: src/app/modules/user/services/user.service.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
```
Comment (scastillo-jp) [inline src/app/modules/user/services/user.service.ts:3]: delete this line

File: src/app/modules/user/services/user.service.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  providedIn: 'root',
+})
+export class UserService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  baseUrl = `${environment.timeTrackerApiUrl}/users`;
+
+  loadUser(userId): Observable<any> {
```
Comment (scastillo-jp) [inline src/app/modules/user/services/user.service.ts:17]: I think can change `Observable<any>`  to `Observable<string>`

File: src/app/app.module.ts (hunk 2 above)
Comment (thegreatyamori) [reply src/app/app.module.ts:165]: Yes, both effects do different things.
```
import { UserEffects as UsersEffects } from './modules/users/store/user.effects';
import { UserEffects } from './modules/users/users/store/user.effects';
```
We found that there is a conflict with the names, especially in the `UserEffects` of users.

Should we change the name?

File: src/app/modules/user/services/user.service.ts (hunk 5 above)
Comment (thegreatyamori) [reply src/app/modules/user/services/user.service.ts:17]: thank you for your observation.
Actually an observable of type `User` is returned.
I willl change it !
🏁

**Review by scastillo-jp — APPROVED** [review]

**Review by kellycastrof — COMMENTED** [review]
A question just to understand, why is there a UserEffect and a UsersEffect that come from the same module?
https://github.com/ioet/time-tracker-ui/blob/872952eee59440b910a96d60cd26cdd1ccaf7339/src/app/app.module.ts#L60-L61

**Comment by thegreatyamori** [comment]
> A question just to understand, why is there a UserEffect and a UsersEffect that come from the same module?
> https://github.com/ioet/time-tracker-ui/blob/872952eee59440b910a96d60cd26cdd1ccaf7339/src/app/app.module.ts#L60-L61
Because there are two nrgx data stores.
The one we made is called `UserEffect` and it is used to query the logged user data.
The first one is `UsersEffects` and it is used to fetch all the registered users. With @Angeluz-07 we decided to add an alias to avoid having to modify the rest of the code since it did not correspond to us.
However, the name should be changed to `UsersEffect` to avoid future conflicts.

**Review by LEON12699 — APPROVED** [review]

**Pull request merged**